boxyhq/saas-starter-kit
# Bugfix - New Team action does not open the modal if it was opened before

### Description
It fixes the bug reported in the issue https://github.com/boxyhq/saas-starter-kit/issues/739 by pushing the route to `/teams` when the New Team modal is closed.

### Bug before the fix
Steps to reproduce:
* Login
* On the sidebar, click on "New Team"
    * It will open the modal
* Close the modal
* On the sidebar, click again on "New Team"
    * It will do nothing

### Validation
Step to validate:
* Login
* On the sidebar, click on "New Team"
    * It will open the modal
* Close the modal
* On the sidebar, click again on "New Team"
    * It will open the modal again, as expected
